testhelp/testhelper.go

```diff
@@ -1,4 +1,4 @@
-package help
+package testhelp
```

Comment:
In Go, a test file must be called `something_test.go`, with the `_test` suffix.

In addition, a test file can have two _package_ names:
- the same name as the package, for example `cogito`. This puts the test code in the same namespace of the SUT, and allows to test non-exported symbols.
- the same name as the package, with a `_test` suffix, for example `cogito_test`. This puts the test code in a different namespace, so that it mimics real client code and has access only to exported symbols of the SUT.

So package name `testhelp` cannot mean that it is used to test the `--help` functionality, although I understand it can be confusing if one doesn't know the Go testing conventions.

If you still find it confusing, I have no problems in renaming, just let me know.

Reply:
Mhh, you put a bug in my hear that it doesn't want to go away :-)
I will rename it.

Reply:
OK. Fine for me and thanks for the explanation. I'm too much used to the python convention of using `test_` as prefix instead of the go suffix way. No need to change if it's the go way. :-)